variables.tf

```
@@ -83,6 +83,12 @@ variable "container_registry_managed_identity_client_id" {
   default     = null
 }
 
+variable "always_on" {
+  description = "Defines the always_on parameter in site_config"
```

Update variable description to better describe what it actually does:

```suggestion
  description = "Should this Web App be loaded even when there is no traffic?"
```